### ISSUE TITLE
Refresh all resources in a project

### ISSUE THREAD

**Issue by imsdu** [BlueBrain/nexus#4104]
**Motivation**
Whenever a `@context` changes for example and as the compacted and expanded forms of a resource are computed, we need  a way to propagate this change to resources

**Acceptance criteria**
1. A user with a dedicated permission can trigger a refresh via the `/v1/projects/{org_label}/{label}/refresh` endpoint
2. When a list of resource types is provided, only the resources associated to those types are updated
3. Deprecated resources are ignored 
4. Only one refresh operation can be run at a time for a given project
5. A refresh operation must survive restarts and save its progress / the errors the same way views do
6. Updates are performed on resources up to the offset where the refresh was triggered
7. When the refresh operation reaches this offset, the related projection stops and the operation is mark as completed

**API changes**
```
PUT /v1/projects/{org_label}/{label}/refresh
{
   "resourceTypes": [ "{resourceType}", ...]
}

**Not in scope (will be tackled in later tickets)** 
1. Listing refresh operations for a project (including completed ones)
2. Fetching the status and statistics of a given refresh operation
3. Fetching errors for a given refresh operation
4. Do not update the resource if the resulting expanded form remains the same

**Comment by crisely09** [comment]
So about 6-7. I guess there are two values, a starting point and a finish point, offset is usually the starting point, would you also have a `limit`/`size` variable?

**Comment by imsdu** [comment]
The last offset can be automatically computed on our side when before starting to update
The limit/size would be handled by configuration
